sound/soc/sof/intel/shim.h

```diff
@@ -178,8 +178,4 @@ extern const struct sof_intel_dsp_desc bdw_chip_info;
 extern const struct sof_intel_dsp_desc hsw_chip_info;
 extern const struct sof_intel_dsp_desc tng_chip_info;
 
-struct sof_intel_stream {
-	size_t posn_offset;
-};
-
```

Comment:
Would the BYT build fail if you remove this without updating intel-ipc.c ? Or was this comptely unused?

Reply:
This was only used by [struct sof_intel_hda_stream](https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/sof/intel/hda.h#L423). So we can remove it if we remove struct sof_intel_stream stream from struct sof_intel_hda_stream. intel-ipc.c define and use struct [intel_stream](https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/sof/intel/intel-ipc.c#L23) locally.

Reply:
@bardliao Ack, thanks for clarifying!